packages/core/workers/src/Worker.js

```diff
@@ -53,10 +53,11 @@ export default class Worker extends EventEmitter {
 
     for (let i = 0; i < filteredArgs.length; i++) {
       let arg = filteredArgs[i];
-      if (
-        (arg === '-r' || arg === '--require') &&
-        filteredArgs[i + 1] === '@parcel/register'
-      ) {
+      let hasArgWithParam =
+        ((arg === '-r' || arg === '--require') &&
+          filteredArgs[i + 1] === '@parcel/register') ||
+        arg === '--title';
```

Comment:
I can make this into a regex (like the single args filtered above) if you'd prefer. It's kind of pointless when there's only one, but I could do it for consistency.